uk.co.piggz.amazfish.appdata.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <name>Amazfish</name>
+  <summary>Smart watch sync app</summary>
+  <id>uk.co.piggz.amazfish</id>
+  <launchable type="desktop-id">harbour-amazfish-ui.desktop</launchable>
```

Comment:
It is flatpak specific. Other distros doesn't require that

Reply:
`rename-desktop-file` in the manifest would take care of that. See the other comment about also using upstream .desktop.

Again. the launchable MUST match the install desktop file name.

Reply:
I mean non-SailfishOS specific. 

In the end Icon and Exec entries are modified, file is renamed. X-Sailjail, X-Nemo-Application-Type are not used in Flatpak. I thought that it would be more understandable to ship it as separate file. Now should conform to flatpak packaging habits.